Makefile

```diff
@@ -37,16 +47,42 @@ TESSERACT_VERSION := 4.1.0
 TESSDATA_REPO = _best
 
 # Ground truth directory. Default: $(GROUND_TRUTH_DIR)
-GROUND_TRUTH_DIR := data/ground-truth
+GROUND_TRUTH_DIR := $(OUTPUT_DIR)-ground-truth
 
 # Max iterations. Default: $(MAX_ITERATIONS)
 MAX_ITERATIONS := 10000
 
+# Debug Interval. Default:  $(DEBUG_INTERVAL)
+DEBUG_INTERVAL := 0
+
 # Network specification. Default: $(NET_SPEC)
 NET_SPEC := [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c\#\#\#]
 
-# Normalization Mode - see src/training/language_specific.sh for details. Default: $(NORM_MODE)
-NORM_MODE = 2
+# Index for Layer to be replaced. Default: $(LAYER_APPEND_INDEX)
+LAYER_APPEND_INDEX := 5
+
+# Replace Layer Network specification. Default: $(LAYER_NET_SPEC)
+LAYER_NET_SPEC := [Lfx192 O1c1]
+
+# Training Build Type - Impact, Plus, Layer or Scratch. Default: '$(BUILD_TYPE)'
+BUILD_TYPE := Scratch
+
+# Language Type - Indic, RTL or blank. Default: '$(LANG_TYPE)'
+LANG_TYPE ?=
+
+# Normalization mode - 2, 1 - for unicharset_extractor and Pass through Recoder for combine_lang_model
+ifeq ($(LANG_TYPE),Indic)
+	NORM_MODE =2
+	RECODER =--pass_through_recoder
+else
+ifeq ($(LANG_TYPE),RTL)
+	NORM_MODE =3
+	RECODER =--pass_through_recoder --lang_is_rtl
+else
+	NORM_MODE =1
```

Comment:
The default for `NORM_MODE` was 2 before. You propose to set it to 1, why?

Reply:
unicharset_extractor
Usage: unicharset_extractor [--output_unicharset filename] [--norm_mode mode] box_or_text_file [...]
Where mode means:
 1=combine graphemes (use for Latin and other simple scripts)
 2=split graphemes (use for Indic/Khmer/Myanmar)
 3=pure unicode (use for Arabic/Hebrew/Thai/Tibetan)
Reads box or plain text files to extract the unicharset.

Reply:
@stweil Which norm_mode are you using for Fraktur training?

Reply:
@wrznr Reverted to 2 via https://github.com/tesseract-ocr/tesstrain/pull/118/commits/9a89d9edfd2161efbe7c441a3e20e3b0ee970813